src/features/contactmoment/service.ts

```diff
@@ -109,6 +132,94 @@ export function useContactverzoekenByKlantId(
   });
 }
 
+export function useContactmomentenByKlantId(
+  id: Ref<string>,
+  page: Ref<number>
+) {
+  function getUrl() {
+    const searchParams = new URLSearchParams();
+    searchParams.set("klant", id.value);
+    return `${klantcontactmomentenUrl}?${searchParams.toString()}`;
+  }
+
+  const mapObjectContactmoment = (a: any) => {
+    return a as ObjectContactmoment;
+  };
+
+  const objectcontactmomenten = async (
+    x: any,
+    objectcontactmomenten: Array<any>
+  ) => {
+    const zaken = [];
+
+    for (const item of objectcontactmomenten) {
+      const relUrl = toRelativeProxyUrl(item, contactmomentenProxyRoot);
+      if (relUrl) {
+        const objectcontactmoment = await fetchLoggedIn(relUrl)
```

Comment:
kan dit met een promise.all?

Reply:
zie de meerwaarde niet in deze situatie. later bespreken